src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -1378,7 +1381,7 @@ private function registerTranslatorConfiguration(array $config, ContainerBuilder
             return;
         }
 
-        $locales = $config['enabled_locales'] ?? [];
+        $locales = $config['enabled_locales'] ?: $enabledLocales;
```

Comment:
shouldn't we also have the comment here about the change needed in 6.0 ?

Reply:
indeed, comment added